test/integration/merchant_account_test.exs

```diff
@@ -0,0 +1,101 @@
+defmodule Braintree.Integration.MerchantAccountTest do
+  use ExUnit.Case, async: true
+
+  @moduletag :integration
+
+  alias Braintree.MerchantAccount
+
+  @master_merchant_id Braintree.get_env(:master_merchant_id)
+
+  describe "create/1" do
+    test "without any params" do
+      assert {:error, :server_error} = MerchantAccount.create()
+    end
+
+    test "with valid params" do
+      assert {:ok, merchant} = MerchantAccount.create(merchant_params())
+
+      refute merchant.id == nil
+    end
+
+    test "with invalid params" do
+      params = %{"tos_accepted" => false, "master_merchant_account_id" => @master_merchant_id}
+
+      assert {:error, error} = MerchantAccount.create(params)
+      assert error.message == "Terms Of Service needs to be accepted. Applicant tos_accepted required."
```

Comment:
This doesn't need to be an exact match of the error message, something like `=~ ~r/terms of service needs to be accepted/i` would make it a little less brittle.